tests/acceptance/src/test/resources/org/activityinfo/test/acceptance/design.feature

```diff
@@ -0,0 +1,12 @@
+@web
+Feature: Design
+  Creation/Design of AI entities
+
+  Background:
+    Given I have created a database "RRMP"
+
+  Scenario: Creation and deletion of location types
+    When I have created a location type "Oblast"
+    Then Location type "Oblast" should appear in tree.
+    When I have removed the location type "Oblast"
+    Then Location type "Oblast" should disappear from tree.
```

Comment:
This is great. 
One additional scenario that need to be covered before it can be released - 
Verify that only users with design permissions on a database can delete the location type